src/main/java/org/folio/consortia/domain/dto/Payload.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.consortia.domain.dto;
+
+import lombok.Data;
+
+@Data
+public class Payload {
+
+  private String user_id;
```

Comment:
should be renamed to userId to satisfy java naming convention, to properly parse from json we can use @JsonProperty("user_id")